core/artwork.qbs

```diff
@@ -103,6 +103,24 @@ Product {
     }
 
     Group {
+        name: "User-role icons"
+        condition: installIcons
+        fileTags: [ "install" ]
+        qbs.installDir: shareDir + "/icons/hicolor/scalable/status/"
+        qbs.install: true
+        files: "../artwork/icons/user-status/hicolor/scalable/status/*.svg"
+    }
+
+    Group {
+        name: "Clients icons"
+        condition: installIcons
```

Comment:
Please add separate boolean property for determining if we should install this icons. There may be license issues so maintainers should be able to disable their installation.